go.mod

```diff
@@ -71,6 +69,7 @@ require (
 	github.com/Azure/go-autorest/autorest/to v0.4.0 // indirect
 	github.com/Azure/go-autorest/autorest/validation v0.3.1 // indirect
 	github.com/Azure/go-autorest/logger v0.2.1 // indirect
+	github.com/Azure/go-autorest/tracing v0.6.0 // indirect
```

Comment:
This might be due to my lack of understanding of `go.mod` files, but shouldn't any mention of this package be removed? I see that it is moved here from above.

Reply:
I thought that was confusing as well, but it persists after `make modules` / `go mod tidy`. I assume it's a dependency of some other package we do import.